### ISSUE TITLE
Matching needs with aids

### ISSUE THREAD

**Issue by nipunadodan** [reliefsupports/reliefsupports.org-v1#126]
Better if can implement a matching mechanism for needs and aids/supports and let each others know there's match for their need or aid.

**Comment by danishka** [comment]
@nipunadodan would you add more details on this please?
right now people can search both requests and donations together at the home page.

for example, search as 'water matara'

But I would like to here more details on your experience.

**Comment by nipunadodan** [comment]
Yeah, I mean some mechanism to notify each party when there's a match from other party...
